Cargo.toml

```diff
@@ -1,25 +1,40 @@
 [package]
 name = "tree-sitter-grep"
-version = "0.1.0"
+version = "0.1.0-dev.0"
 edition = "2021"
 license = "Unlicense OR MIT"
+authors = [
+  "Julian Rosse <julian@helixbass.net>",
+  "Peter Stuart <peter@peterstuart.org>"
+]
+description = """
+tree-sitter-grep is a grep-like search tool that
+recursively searches the current directory for a
+tree-sitter query pattern. Like ripgrep, it respects
+gitignore rules.
+"""
+homepage = "https://github.com/helixbass/tree-sitter-grep"
+repository = "https://github.com/helixbass/tree-sitter-grep"
+keywords = ["tree-sitter", "grep", "search"]
+categories = ["command-line-utilities", "development-tools"]
+rust-version = "1.70"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
 bstr = "1.1.0"
 bytecount = "0.6"
-clap = { version = "4.3.0", features = ["derive"] }
+clap = { version = "4.3.0", features = ["derive", "wrap_help"] }
 encoding_rs = "0.8.14"
 encoding_rs_io = "0.1.6"
-ignore = { git = "https://github.com/helixbass/ripgrep", rev = "012f2f1" }
+ignore = { package = "tree_sitter_grep_ignore", git = "https://github.com/helixbass/ripgrep", rev = "669ebd3", version = "0.4.20-dev.0" }
```

Comment:
From reading a couple different things all of your dependencies have to be published (vs eg `git` dependencies) in order for you to publish

So I updated the package names/descriptions of these forked/local dependencies (to have a `tree_sitter_grep_*` prefix) and published them as `whatever-dev.0` versions

At that point I was able to go ahead and publish a `0.1.0-dev.0` version